examples/container-deny-added-caps/src_test.rego

```diff
@@ -1,22 +1,13 @@
 package container_deny_added_caps
 
 test_dropped_all {
-    input := {
-        "securityContext": {
-            "capabilities": {"drop": ["alL"]}
```

Comment:
@jalseth is the capitalization of "alL" made on purpose or should it rather be "all"?

Reply:
This should be "all"

Reply:
Thanks, I changed it and regenerated the policies so the acceptance tests should work.

Reply:
It was actually intentional to verify the policy to ignore the casing of the key word, but it really doesn't matter at this point :)